components/cam/src/physics/cam/shoc.F90

```diff
@@ -1751,10 +1751,10 @@ subroutine f0_to_f5_diag_third_shoc_moment(&
   wsec_diff     = w_sec_kc    - w_sec
   tke_diff      = tke_kc      - tke
 
-  f0 = thedz2 * bet2**3 * iso**4 * wthl_sec * &
+  f0 = thedz2 * bfb_cube(bet2) * bfb_pow(iso,4._rtype) * wthl_sec * &
```

Comment:
I guess this call to `bfb_pow` may change once we get `pow<4>` into EKAT. Just a comment--no need to change anything.

Reply:
Ah, never mind. :-) Andrew's got the in-time solution.